roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1,6 @@
-NETPLUGIN_ARGS=""
+{# If an interface is specified for netplugin, use it #}
+{% if netplugin_if is defined %}
+NETPLUGIN_ARGS='-docker-plugin -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+{% else %}
+NETPLUGIN_ARGS='-docker-plugin -vtep-ip {{node_addr}}'
```

Comment:
I think as we debugged this will break non-aci setups, we should just require `netplugin_if` to be defined always

Just following shall be enough:

```
NETPLUGIN_ARGS='-docker-plugin -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
```